core/server/services/auth/authenticate.js

```diff
@@ -38,6 +39,13 @@ const authenticate = {
             req.body.client_secret = req.query.client_secret;
         }
 
+        if (labs.isSet('publicAPI') !== true) {
```

Comment:
We already handle this in

> core/server/services/auth/authorize.js

What was the decision to not change the existing place?

Reply:
Ok 👍